workitem_blackbox_test.go

```diff
@@ -84,6 +86,66 @@ func TestGetWorkItem(t *testing.T) {
 		t.Errorf("expected title %s, but got %s", "Updated Test WI", updated.Data.Attributes[workitem.SystemTitle])
 	}
 
+	_, result2 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload)
+	_, result3 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload)
+	r2, _ := strconv.Atoi(fmt.Sprintf("%v", *result2.Data.ID))
+	r3, _ := strconv.Atoi(fmt.Sprintf("%v", *result3.Data.ID))
+	payload2 = minimumRequiredUpdatePayload()
+	payload2.Data.ID = updated.Data.ID
+	payload2.Data.Attributes = updated.Data.Attributes
+	payload2.Data.Attributes["previousitem"] = r2
+	payload2.Data.Attributes["nextitem"] = r3
+	_, updated1 := test.UpdateWorkitemOK(t, nil, nil, controller, *updated.Data.ID, &payload2)
+	assert.NotNil(t, updated1.Data.Attributes[workitem.SystemCreatedAt])
+	assert.NotNil(t, updated1.Data.Attributes[workitem.SystemOrder])
+
+	if updated1.Data.Attributes["version"] != (updated.Data.Attributes["version"].(int) + 1) {
```

Comment:
Since you've already been using `assert` , you could use them in this format `assert.NotEqual(t, obj1, obj2, "two objects shouldn't be equal")`